File: kotlin/challenges/checkingSecurityURL.kt
fun main() {
    val url = readLine() ?: "" 

    val resultado: String

    if (url.startsWith("https://")) {
        resultado = "URL segura"
    } else if (url.startsWith("http://")) {
        resultado = "URL nao segura"
    } else {
      resultado = "Formato invalido"
    }
    println(resultado)
}

File: c#/challenges/checkingSecurityURL.cs
using System;

class Program {
    static void Main(string[] args) {
        string url = Console.ReadLine(); 
        string resultado;
        if (url.StartsWith("https://")) {
          resultado = "URL segura";
        } else if (url.StartsWith("http://")) {
          resultado = "URL nao segura";
        } else {
          resultado = "Formato invalido";
        }
        Console.WriteLine(resultado);
    }
}